src/com/lilithsthrone/game/character/effects/StatusEffect.java

```diff
@@ -5303,8 +5303,8 @@ public String extraRemovalEffects(GameCharacter target) {
 		@Override
 		public boolean isConditionsMet(GameCharacter target) {
 			float cumAmount = target.getTotalFluidInArea(SexAreaOrifice.ANUS) + target.getTotalFluidInArea(SexAreaOrifice.MOUTH) + target.getTotalFluidInArea(SexAreaOrifice.VAGINA);
-			return cumAmount >= CumProduction.SEVEN_MONSTROUS.getMedianValue()
-					&& cumAmount < CumProduction.SEVEN_MONSTROUS.getMaximumValue()
+			return cumAmount >= Math.pow((Math.cbrt(7570) / 182.88) * target.getHeightValue(), 3)
```

Comment:
This calculation is almost equal to `Math.pow(target.getHeightValue(), 3) / 808` (~0.002% difference) and is a much simpler calculation.